certbot/main.py

```diff
@@ -413,6 +414,9 @@ def _find_domains_or_certname(config, installer, question=None):
     # if we can't do that but we have a certname, get the domains
     # with that certname
     elif certname:
+        if bool(re.search(r'[\\/]', certname)) or bool(re.search('^-', certname)):
+            raise errors.Error("A certname cannot contain slashes (/ or \) "
```

Comment:
I think the backslash in here is likely to give trouble, probably want to `\\` that.